packages/strapi-plugin-content-type-builder/admin/src/containers/DataManagerProvider/index.js

```diff
@@ -59,7 +60,10 @@ const DataManagerProvider = ({ allIcons, children }) => {
   const componentMatch = useRouteMatch(
     `/plugins/${pluginId}/component-categories/:categoryUid/:componentUid`
   );
-  const isInDevelopmentMode = currentEnvironment === 'development';
+  const formatMessageRef = useRef();
```

Comment:
Is the `useRef()` really it necessary here?

Reply:
Yes so this way we don't have to pass it in the dependencies array of the `useEffect` hook so it avoids the stale closure issue.